modules/rest-api/pages/rest-bucket-create.adoc

```diff
@@ -19,7 +19,7 @@ POST /pools/default/buckets/<bucketName>
 == Description
 
 These respectively create a new bucket and edit an existing bucket.
-The bucket can be of any type: Couchbase, Ephemeral, or Memcached.
+The bucket can be of any type: Couchbase, Ephemeral, or Memcached. 
```

Comment:
Watch out for odd spaces being added to the end of lines. It causes false changes to show up in the version control.